packages/webview_flutter/webview_flutter/android/src/main/java/io/flutter/plugins/webviewflutter/FlutterWebView.java

```diff
@@ -258,6 +270,14 @@ private void loadUrl(MethodCall methodCall, Result result) {
     result.success(null);
   }
 
+  public void postUrl(MethodCall methodCall, Result result) {
```

Comment:
I think this method should be `private`, if there is a reason to keep it `public` it should be documented.